setup.py

```diff
@@ -18,8 +18,8 @@
 
 tests_require = [
     'pytest-mock>=1.6.0',
-    'pytest-invenio>=1.4.1',
-    'invenio-accounts>=1.4.3',
+    'pytest-invenio>=1.4.13,<2.0.0',
+    'invenio-accounts>=2.0.0',
```

Comment:
minor: Do we want to cap this one, given it's part of an LTS?

Reply:
Yes, it is for the tests, it is needed otherwise they don't pass.
It does not affect the installation of the module.

Reply:
Sorry that was unclear, what I meant was:

Shouldn't we do:

'invenio-accounts>=2.0.0,<3.0.0',

given so that the LTS is resilient to major upgrade bumps by this dependency? 

This can be addressed in a separate PR :+1: